server/java/README.md

```diff
@@ -0,0 +1,75 @@
+# Stripe Payments Demo - Java Server
+
+This demo uses a simple [Spark](http://sparkjava.com) application as the server.
+
+## Payments Integration
+
+- [`Application.java`](src/main/java/app/Application.java) contains the routes that interface with Stripe to create charges and receive webhook events.
+
+## Requirements
+
+You’ll new the following:
+
+- [Java 8](https://www.oracle.com/technetwork/java/javase/overview/java8-2100321.html)
+- [Gradle](https://gradle.org/)
+- Modern browser that supports ES6 (Chrome to see the Payment Request, and Safari to see Apple Pay).
+- Stripe account to accept payments ([sign up](https://dashboard.stripe.com/register) for free!)
+
+## Getting Started
+
+Before getting started check that you have java installed
+
+```
+java -version
+```
+
+Set the enviroment variables for the web application. You can copy the environment variables from [here](../../.env.example)
+
+```
+export STRIPE_PUBLISHABLE_KEY=
+export STRIPE_SECRET_KEY=
+export STRIPE_WEBHOOK_SECRET=
+export STRIPE_ACCOUNT_COUNTRY=
+export PAYMENT_METHODS=
+export NGROK_SUBDOMAIN=
+export NGROK_AUTHTOKEN=
+```
+
+Use `gradle` to install the required dependencies by navigating to ./server/java and running:
+
+```
+gradle build
+```
+
+Use `gradle` to build a "fatjar" including all the apps dependencies.
+
+```
+gradle shadowJar
+```
+
+This will build a JAR in the `./build/libs` directory. Running this JAR will start the Spark Web Application on port 4567.
+
+```
+java -jar build/libs/salesPaymentDemo-1.0-SNAPSHOT.jar
```

Comment:
```
$ java -jar build/libs/salesPaymentDemo-1.0-SNAPSHOT.jar
Exception in thread "main" java.lang.NoClassDefFoundError: spark/Spark
        at app.Application.main(Application.java:13)
Caused by: java.lang.ClassNotFoundException: spark.Spark
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 1 more
```

Reply:
Tried to reproduce with Karl and his worked. When you are online we can investigate